kyuubi-common/src/main/scala/org/apache/kyuubi/engine/deploy/yarn/EngineYarnModeSubmitter.scala

```diff
@@ -92,6 +94,28 @@ abstract class EngineYarnModeSubmitter extends Logging {
   def engineExtraJars(): Seq[File] = Seq.empty
 
   protected def submitApplication(): Unit = {
+    val user = kyuubiConf.getOption(KYUUBI_SESSION_USER_KEY)
```

Comment:
we actually have seen that limitation of always using session user as the engine launch user, let's define a dedicated `proxyUser: Option[String]`

Reply:
The user retrieved in this context is identified as appUser, which is supplied via the ProcessBuilder using the --conf parameter. However, the naming of the configuration key associated with this user appears to be somewhat unclear. 

Are you suggesting that the key name should be updated for clarity, or are you indicating that appUser does not represent the intended proxy user?